src/main/java/frc/robot/drive/MotorType.java

```diff
@@ -0,0 +1,7 @@
+package frc.robot.drive;
+
+public enum MotorType {
+    TALON_SRX,
+    SPARK_MAX_BRUSHED,
+    SPARK_MAX_BRUSHLESS
```

Comment:
What is the difference between these 3?

Reply:
Talked to the mentor about this, figured out the difference (talon is older and brushless, spark max is newer and comes in 2 types)